flocker/control/_model.py

```diff
@@ -418,18 +416,16 @@ def __invariant__(self):
                     return (False, '%r manifestation is not on node' % (app,))
         return (True, "")
 
-    def __new__(cls, **kwargs):
+    def __new__(cls, hostname=None, **kwargs):
         # PRecord does some crazy stuff, thus _precord_buckets; see
         # PRecord.__new__.
         if "uuid" not in kwargs and "_precord_buckets" not in kwargs:
             warn("UUID is required, this is for backwards compat with existing"
                  " tests only. If you see this in production code that's "
                  "a bug.", DeprecationWarning, stacklevel=2)
```

Comment:
Well can't we just look for these warnings when running the test suite, fix those tests and then remove this. Or are we planning to do that in a followup issue. If so which issue?